src/Raven.Studio/typescript/commands/database/index/clearIndexErrorsCommand.ts

```diff
@@ -0,0 +1,17 @@
+import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+
+class clearIndexErrorsCommand extends commandBase {    
+    constructor(private indexName: string, private db: database) {
```

Comment:
this endpoint allows you do specify list of indexes to clean, so method signature should be:
indexNames: string[]; db: database

Reply:
done